<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCreate_A_New_Shape">
    <property name="title">
     <string>Create A New Shape</string>
    </property>
    <addaction name="actionNew_Shape"/>
   </widget>
   <widget class="QMenu" name="menuDelete_A_Shape">
    <property name="title">
     <string>Delete A Shape</string>
    </property>
    <addaction name="actionDelete_Shape"/>
   </widget>
   <widget class="QMenu" name="menuMove_A_Shape">
    <property name="title">
     <string>Move A Shape</string>
    </property>
    <addaction name="actionMove_Shape"/>
   </widget>
   <addaction name="menuCreate_A_New_Shape"/>
   <addaction name="menuDelete_A_Shape"/>
   <addaction name="menuMove_A_Shape"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Shape">
   <property name="text">
    <string>New Shape</string>
   </property>
  </action>
  <action name="actionDelete_Shape">
   <property name="text">
    <string>Delete Shape</string>
   </property>
  </action>
  <action name="actionMove_Shape">
   <property name="text">
    <string>Move Shape</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
